jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/java/javacore/JCJavaHeap.java

```diff
@@ -0,0 +1,82 @@
+/*[INCLUDE-IF JAVA_SPEC_VERSION >= 8]*/
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
```

Comment:
Please update this URL in all new files to reflect changes in #16306; it should be
> https://openjdk.org/legal/assembly-exception.html

Reply:
I have updated all the new files.